File: FastIO/FastIO.py
# FastIO.py

# Python의 Fast Input은 sys module의 readline 함수 이용
import sys

input = sys.stdin.readline

# 이후는 동일!

a = input() # 한개의 값 입력 받을 때

n, m = map( int, input().split() ) # "1 2" 이런식으로 한줄에 여러 값이 들어올 때 각각의 변수로 매칭

arr = list( map( int, input().split() ) ) # "1 2 3 4 5" 이런식으로 한줄에 배열이 들어올 때 list로 변경

# Python의 Fast output은 없음! 만약 시간초과가 뜬다면 Pypy3로 제출해보시길...
print( a )

print( n, m )

print ( arr )

File: FastIO/FastIO.cpp
// FastIO.cpp
#include <iostream>

using namespace std;

int main(void) {
	ios_base::sync_with_stdio(false);
	cin.tie( NULL ); // C++의 Fast Input
	cout.tie( NULL ); // C++의 Fast Output

	// 만약 위의 것들을 선언했다면 printf, scanf등 사용 불가!
	// 이후는 동일
	int a;
	cin >> a;

	int n, m;
	cin >> n >> m;

	int arr[10];
	for ( int i = 0; i < 10; i++ )
		cin >> arr[i];

	// 출력 때 개행문자를 endl로 하는 것은 굉장히 느림! 따라서 빠른 Output 때는 꼭 '\n'으로 개행해주기
	cout << a << '\n';

	cout << n << ' ' << m << '\n';

	for ( int i = 0; i < 10; i++ )
		cout << arr[i] << ' ';
	cout << '\n';

	return 0;
}